docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -187,7 +184,7 @@ cy.get('.todo-list li') // query
 />
 
 Only queries can be retried, but most other commands still have built-in
-_waiting_. For example, as described in the "Assertions" section of
+assertions. For example, as described in the "Assertions" section of
```

Comment:
From a marketing perspective, I do think keeping references to auto-awaiting is good buzz words to keep in here even if implicit assertions are how we accomplish this

Reply:
Done.